data/daemon.conf

```diff
@@ -48,3 +48,7 @@ IgnorePower=false
 
 # Only support installing firmware signed with a trusted key
 OnlyTrusted=true
+
+# these are only required when the SMBIOS or Device Tree data is invalid or missing;
+# the full list can be found here: https://fwupd.github.io/libfwupdplugin/
```

Comment:
why not just list them all?  This is (relatively) static and there are only 13 of them.

Reply:
I think it's actually better than that; we only want to override the non-version ones. Patch updated.